### ISSUE TITLE
Latest videos on front page not updating

### ISSUE THREAD

**Issue by para2022** [WWBN/AVideo#8536]
Most recent uploads are not updating on front page. They update immediately with the cache off but with the cache on they only update after the cache expires.

**Comment by DanielnetoDotCom** [comment]
Try this update

**Comment by para2022** [comment]
Fixed buddy. Thank you :)

**Comment by para2022** [comment]
Still a bit of a bug where some videos dont move along in the latest videos. Some videos still stay as the most recent upload even though they were uploaded hours ago and the newer ones are below it instead of above it. When it happens again il send an image so you understand more.

**Comment by DanielnetoDotCom** [comment]
Ok, also check the created date of them

**Comment by para2022** [comment]
will do :)

**Comment by para2022** [comment]
Sorry dan, the same bug is affecting promoted videos also. The views are not updating on main page until a cache update. Thanks

**Comment by DanielnetoDotCom** [comment]
The first page cache should update every 1 min 

Can you confirm that ?

**Comment by para2022** [comment]
Yes that seems be right. Doesn't seem to be much of a problem to be fair. 1 minute update is fine :)
Thanks
